core-schema/gcn.notices.core.HardnessRatio.json

```diff
@@ -1,19 +1,18 @@
 {
+  "$id": "https://github.com/nasa-gcn/gcn-schema/blob/master/core-schema/gcn.notices.core.HardnessRatio.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
   "type": "object",
   "title": "gcn.notices.core.HardnessRatio",
   "description": "Hardness Ratio parameterization",
   "properties" : {
     "hardness_ratio": {
-      "type": ["number"], 
+      "type": "number", 
       "description": "ratio of flux between high and low energy bands"
     },
-    "energy_range_hard": {
-      "$ref": "gcn.notices.core.HardnessRatio",
-      "description": "Energy range [keV] used in hard band of hardness ratio"
-    },
-    "energy_range_soft": {
-      "$ref": "gcn.notices.core.HardnessRatio",
-      "description": "Energy range [keV] used in soft band of hardness ratio"
+    "energy_range": {
+      "$ref": "gcn.notices.core.EventInfo.json",
```

Comment:
By mistake have called "EventInfo" instead of EnergyRange

Reply:
OK. I've approved this. Merge it whenever you are ready.